### ISSUE TITLE
Support a "Publish" option like the website

### ISSUE THREAD

**Issue by leighmcculloch** [hediet/vscode-drawio#248]
This drawio extension is amazing. One feature that it is missing from the drawio website is the ability to publish to a link. The publish link option shows up in the web version here:

<img width="429" alt="Screen Shot 2021-07-12 at 11 46 38 AM" src="https://user-images.githubusercontent.com/351529/125340108-ef5b9e80-e306-11eb-907b-18b97045b14c.png">

The publish link option doesn't actually publish anything to the internet as best I can tell, instead it encodes the diagram into a URL. This is similar to how the embed options work in the extension, but instead of an embed, it is just a URL that is returned.

**Comment by hediet** [comment]
Do you use this feature often? What is the use case for it?

**Comment by leighmcculloch** [comment]
Yes frequently. The publish feature is very helpful when working on a diagram locally and wanting to quickly share it with a colleague. Grabbing a URL is fast.

**Comment by hediet** [comment]
I'll see what I can do.

Btw for sharing, you can also use liveshare!

**Comment by leighmcculloch** [comment]
> Btw for sharing, you can also use liveshare!

There true! But that's slower and not everyone uses VSCode 😄
